conf/i18n/translatehelpervisitor.js

```diff
@@ -36,10 +36,12 @@ class TranslateHelperVisitor {
    * Returning undefined leaves the node unaffected, otherwise it replaces it with the
    * returned value.
    *
-   * @param {hbs.AST.MustacheStatement} statement
-   * @returns {hbs.AST.MustacheStatment|undefined} Either the new node, or undefined to leave the node as is
+   * @param {hbs.AST.MustacheStatement} handlebarsStatement
+   * @returns {hbs.AST.MustacheStatement|undefined} Either the new node, or undefined to leave the node as is
    */
-  _handleMustacheStatement (statement) {
+  _handleMustacheStatement (handlebarsStatement) {
+    const statement = handlebarsStatement;
+    statement.escaped = false;
```

Comment:
this is cool I didn't know you could do this! 
nit but can we move the `statement.escaped = false` to after the translationHelper check? Otherwise it's just slightly less clear that only translation helpers are being affected. could we also take out the `const statement = handlebarsStatement` and just rename statement?

Reply:
Oh true, moved! I had the new variable because I was being paranoid about mutating the input value. Since the MustacheStatement is an Object, I do need to make a new variable, right?

Reply:
Oh, since it's an object making a new variable just makes a "pointer" to the original data, so you'll still be mutating it. It's okay though because this method is mutating the tree anyways.

Reply:
PR lgtm besides this!

Reply:
Ah true because it's by reference, gotcha! Updating this to remove new variable.